packages/contracts/scripts/randomizer-deployments/shared-randomizer-deployer.ts

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import { ethers } from "hardhat";
+import { tryVerify } from "../util/verification";
+import fs from "fs";
+import path from "path";
+
+// hide nuisance logs about event overloading
+import { Logger } from "@ethersproject/logger";
+Logger.setLogLevel(Logger.levels.ERROR);
+
+// delay to avoid issues with reorgs and tx failures
+import { delay, getConfigInputs } from "../util/utils";
+import { EXTRA_DELAY_BETWEEN_TX } from "../util/constants";
+
+/**
+ * This script was created to deploy a shared randomizer contract.
+ * It is intended to document the deployment process and provide a reference
+ * for the steps required to deploy the shared randomizer contract.
+ */
+//////////////////////////////////////////////////////////////////////////////
+// CONFIG BEGINS HERE
+//////////////////////////////////////////////////////////////////////////////
+const randomizerName = "SharedRandomizerV0";
+// if the following is undefined, a new pseudorandomAtomicContract will be deployed
+// if the following is defined, the existing pseudorandomAtomicContract will be used
+let pseudorandomAtomicContractAddress = undefined;
```

Comment:
nit
```suggestion
const pseudorandomAtomicContractAddress = undefined;
```

Reply:
oops removed this locally defined config altogether 😅